scripts/dev-orb-setup/fuse-orb-id.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+set -o errexit
+set -o errtrace
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+BASE_DIR=$(cd -- "$(dirname -- "${SCRIPT_DIR}")" &> /dev/null && pwd)
+
+usage() {
+    echo "Description: Script to fuse a random ORB-ID"
+    echo "Usage: $0 <orb-ip-address>"
+    echo ""
+    echo "Arguments:"
+    echo "  <orb-ip-address>    The IP address of the orb."
+}
+
+generate_random_orb_id() {
+    openssl rand -hex 4
+}
+
+main() {
+    local arg
+    local positional_args=()
+
+    while [[ $# -gt 0 ]]; do
+        arg="${1}"; shift
+        case ${arg} in
+            -h | --help)
+                usage
+                exit 0
+                ;;
+            -*)
+                echo "Invalid argument: ${arg}"
+                usage
+                exit 1
+                ;;
+            *)
+                positional_args+=("${arg}")
+                ;;
+        esac
+    done
+
+    set -- "${positional_args[@]}"
+    if [[ $# -ne 1 ]]; then
+        echo "Error: Exactly one positional argument is required."
+        usage
+        exit 1
+    fi
+
+    local orb_ip="${1}"
+    local lock_status
+    local orb_id
+
+    ssh -M -S tmp-ssh-socket -fN worldcoin@"${orb_ip}"
+
+    lock_status=$(ssh -S tmp-ssh-socket worldcoin@"${orb_ip}" 'cat /sys/devices/platform/tegra-fuse/odm_lock')
```

Comment:
Nit: All the `ssh` commands can fail in cases related to the number of keys configured on the host.

Because you're using the same socket, I _**think**_ it's already authed and you don't need to reauth on each subsequent command.

Reply:
you mean I can omit the `worldcoin@"${orb_ip}"` in all of the `ssh` commands?